pipeline/experiment.py

```diff
@@ -277,6 +310,63 @@ class ActionEvent(dj.Imported):
     action_event_time : decimal(8,4)  # (s) from trial start
     """
 
+
+# ---- Foraging paradigm specifics ----
+
+@schema
+class SessionBlock(dj.Imported):
+    definition = """
+    -> Session
+    block : smallint 		# block number
+    ---
+    block_uid : int  # unique across sessions/animals 
+    block_start_time : decimal(10, 4)  # (s) relative to session beginning
+    """
+
+    class WaterPortRewardProbability(dj.Part):
+        definition = """
+        -> master
+        -> WaterPort
+        ---
+        reward_probability: decimal(8, 4)
+        """
+
+    class BlockTrial(dj.Part):
+        definition = """
+        -> master
+        -> BehaviorTrial
+        """
+
+
+@schema
+class WaterValveSetting(dj.Imported):
```

Comment:
We should probably use WaterPortSettings/WaterPortInfo/WaterPortData instead to make it similar to other table names, in agreement with Han's comment, and using port instead of valve